include/llvm/Support/ManagedStatic.h

```diff
@@ -1,9 +1,8 @@
 //===-- llvm/Support/ManagedStatic.h - Static Global wrapper ----*- C++ -*-===//
 //
-//                     The LLVM Compiler Infrastructure
-//
-// This file is distributed under the University of Illinois Open Source
-// License. See LICENSE.TXT for details.
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
I appreciate that this is coming from a cherry-pick, but changing the file license is... not something we can do in a cherry-pick.

LLVM's code is currently dual licensed, so it is still also available under the UIUC open source license which is what DXC's code is mostly under.